CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.406 Dec 18 2024
+- Expose the GCP Shielded VM setting at the machine pool level
```

Comment:
Can we specify that it is about the SecureBoot feature part of GCP Shielded VMs ?

Reply:
addressed.